app/deliver_grant_funding/routes.py

```diff
@@ -46,6 +47,8 @@ def grant_setup_ggis() -> ResponseReturnValue:
         grant_session.has_ggis = form.has_ggis.data
         grant_session.ggis_number = form.ggis_number.data if form.has_ggis.data == "yes" else None
         session["grant_setup"] = grant_session.to_session_dict()
+        if request.args.get("return_to_check"):
```

Comment:
I added some behaviour like this in form runner recently, but used the query arg `source`. Could we use that here as well please?

I think we'll soon enough need some service-wide way of handing/directing backlinks, which is bubbling in the back of my head slightly.

I don't think we need to do a service framework here (but we do have [a ticket for it](https://mhclgdigital.atlassian.net/browse/FSPT-417)), so I think let's just be manually consistent for now, I think?

I don't think you need the enum for values, maybe just define a CHECK_YOUR_ANSWERS='check-your-answers' constant or something. Or you could just use the presence of the `source` kwarg as an empty string, like you do here, but that feels a pinch less readable in this case.